src/index.css

```
@@ -296,15 +296,15 @@
   /* 高亮动画关键帧 */
   @keyframes highlight-pulse {
     0% {
-      background-color: var(--background-color-alpha);
+      /*background-color: var(--background-color-alpha);*/
```

Stylelint (17.11.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)